core/templates/dev/head/css/oppia.css

```diff
@@ -702,6 +702,7 @@ textarea {
   Styles for footer
 */
 .oppia-footer {
+  position: relative !important;
```

Comment:
We try to avoid using the important hack. Can you find a way to implement this without that? Also please put your CSS properties in alphabetical order.